src/nupic/algorithms/SpatialPooler.cpp

```diff
@@ -1183,7 +1192,7 @@ void SpatialPooler::load(istream &inStream) {
   // Check the saved version.
   UInt version;
   inStream >> version;
-  NTA_CHECK(version <= version_);
+  NTA_CHECK(version == version_);
```

Comment:
So, you are not going to be backward compatible with the previous version?  I guess that is reasonable.

Reply:
good spot! Yeah.. here it would be possible to support the old format, but I've decided to prefer clean,small codebase (not many people actually relying on our code now) Do you agree with the removal?